docs/en/cpp/api_reference/structmavsdk_1_1_info_1_1_identification.md

```diff
@@ -1,39 +1,39 @@
-# mavsdk::Info::Identification Struct Reference

```

Comment:
Shouldn't all this ref be updated/removed/modified by CI?

Reply:
Yes, I'll look into that.